public/Day-221/scripts/ui/editor.js

```
@@ -0,0 +1,375 @@
+
+                    for (let i = 0; i < 10; i++) {
+                        this.addNode(
+                            clusterX + (Math.random() - 0.5) * 100,
+                            clusterY + (Math.random() - 0.5) * 100
+                        );
+                    }
+                }
+                break;
+
+            case 'circle':
+                const radius = 200;
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)